talisker/gunicorn.py

```diff
@@ -179,6 +194,24 @@ def init(self, parser, opts, args):
 
         return cfg
 
+    def load_config(self):
+        super(TaliskerApplication, self).load_config()
+
```

Comment:
I'm not sure about b); I'd rather let users override the hook completely and do whatever they want with it.